ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Examples.hs

```diff
@@ -94,9 +94,10 @@ import           Test.Shelley.Spec.Ledger.Orphans ()
 import qualified Test.Shelley.Spec.Ledger.Utils as SL hiding (mkKeyPair,
                      mkKeyPair', mkVRFKeyPair)
 
+import           Ouroboros.Consensus.Shelley.Eras (StandardShelley)
```

Comment:
Plan for #2669:
* Make the examples monomorphic
* Each Shelley-flavoured era can include examples from the previous era where applicable, using the translation infrastructure from the ledger where possible (or another injection function).